tests/verifyRemoteInstances_test.js

```diff
@@ -208,6 +208,35 @@ Scenario(
     await pmmInventoryPage.checkAgentOtherDetailsSection('listen_port:', `listen_port: ${remoteInstancesHelper.remote_instance.haproxy.haproxy_2.port}`, haproxyServiceName, serviceId);
   },
 );
+Scenario(
+  'PMM-T1089 - Verify UI elements for PostgreSQL Instance @instances',
+  async ({
+    I, remoteInstancesPage,
+  }) => {
+    I.amOnPage(remoteInstancesPage.url);
+    remoteInstancesPage.waitUntilRemoteInstancesPageLoaded();
+    remoteInstancesPage.openAddRemotePage('postgresql');
+    I.click(remoteInstancesPage.fields.addService);
+    remoteInstancesPage.checkRequiredField();
+    // Verify fields on the page
+    I.waitForVisible(remoteInstancesPage.fields.returnToMenuButton, 30);
```

Comment:
waitFor is not wrong, but ideally, I would use I.seeElement, and only use wait for visible when I want to either ensure that the UI component loaded successfully, so here just one WaitForVisible when the Page for adding pgsql opens up and then I.seeElement's